drivers/bbram/bbram_it8xxx2.c

```diff
@@ -91,12 +105,16 @@ static int bbram_it8xxx2_init(const struct device *dev)
 }
 
 #define BBRAM_INIT(inst)                                                                           \
-	static const struct bbram_it8xxx2_config bbram_cfg_##inst = {                              \
-		.base_addr = DT_INST_REG_ADDR(inst),                                               \
-		.size = DT_INST_REG_SIZE(inst),                                                    \
-	};                                                                                         \
+	BBRAM_DECL_CONFIG(inst);                                                                   \
```

Comment:
This macro is specific to each chipset.  Please make the names unique.

Reply:
Done